src/subcommands/create_csr.rs

```diff
@@ -181,11 +181,18 @@ impl CreateCsr {
                     // Key policy specifies raw RSA signatures. RCGEN will always hash-and-sign, so fail.
                     error!("Signing key specifies raw signing only, which is not supported for certificate requests.");
                     Err(ToolErrorKind::NotSupported.into())
-                }
-                AsymmetricSignature::RsaPss { .. } => {
-                    error!("Signing key specifies RSA PSS scheme, which is not supported for certificate requests.");
```

Comment:
Why was this not being supported for certificate requests before?

Reply:
This was just not implemented as they didn't need it.